drivers/volume/aws/aws.go

```diff
@@ -672,13 +673,17 @@ func (a *aws) IsVirtualMachineSupported() bool {
 }
 
 func init() {
-	a := &aws{}
-	err := a.Init(nil)
-	if err != nil {
-		logrus.Debugf("Error init'ing aws driver: %v", err)
-	}
-	if err := storkvolume.Register(storkvolume.AWSDriverName, a); err != nil {
-		logrus.Panicf("Error registering aws volume driver: %v", err)
+	if os.Getenv("SKIP_AWS_DRIVER_INIT") == "true" {
```

Comment:
The changes look good. Had a query which out of context of this PR but is there any specific reason to use `init` function here? Pointing out since in Go its not considered a good practice nowadays. Should we remove it from here or will it be an unnecessary change?

Reply:
Just checked the code and found that it is a practice which we are following for all the drivers to register them before stork starts (they are all imported in `cmd/stork/stork.go`). Looks like then its a valid case for using `init` function. Can ignore this review comment.